CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- [#12074](https://github.com/MetaMask/metamask-extension/pull/12074): Remove extra zero balance account sometimes when importing wallet from recovery seed
```

Comment:
note: the changelog is something we update when a revision makes it to a release candidate

Reply:
oh makes sense, reverted the commit to update the changelog. thanks!